conanfile.py

```diff
@@ -392,7 +393,7 @@ def build(self):
         else:
             args.insert(0, "-shared")
         if self.options.multiconfiguration:
-            args.append("-debug-and-release")
+            args.append("-release")
```

Comment:
This is a hack to force conan to configure Qt to build with the Qt provided 3rd party libraries for any missing system libraries.  Enabling multiconfiguration defines the `-qt-<system_lib>` option.  However, you cannot build `-debug-and-release` for linux so I switched it to release as a work around.